lib/src/screens/list_item/list_item_view.dart

```diff
@@ -0,0 +1,70 @@
+import 'package:dio/dio.dart';
+import 'package:flutter/material.dart';
+import 'package:web_provise_assignment/src/model/item_model.dart';
+
+class ListItem extends StatelessWidget {
+  ListItem({super.key});
+  final dio = Dio();
```

Comment:
- `Dio` instance here will create a new instance when this widget  `ListItem` is re-rendered from outside since it's a stateless widget. It's more proper to convert this widget to stateful widget and initialize in stateful widget so it will only create once.
```suggestion
  class ListItem extends StatefulWidget {
  const ListItem({super.key});

  @override
  State<ListItem> createState() => _ListItemState();
}

class _ListItemState extends State<ListItem> {
  final dio = Dio();
  .....
```